examples/4-integrations/3-svelte/src/vite-env.d.ts

```diff
@@ -0,0 +1,2 @@
+/// <reference types="svelte" />
```

Comment:
Is this file used? I don't have it when I create a new project using Vite.

Reply:
I do get these with the generation. But I have no idea what it does. Seems like IntelliSense definitions.
https://v2.vitejs.dev/guide/env-and-mode.html#env-files